p4-16/psa/PSA.mdk

```diff
@@ -921,6 +921,9 @@ in such a way that they return a `GREEN` (also known as conform),
 conditions under which one of these three results is returned.  The P4
 program is responsible for examining that returned result, and making
 changes to packet forwarding behavior as a result.
+The value returned by an uninitialized meter is undefined. If a target
+implementation chooses to return some value, the PSA recommends the
+value should be `GREEN`.
```

Comment:
not sure how relevant this it to you, but we plan in `P4Runtime` to _require_ the default value to be `GREEN`, at least for _direct_ meters.

Reply:
Thanks for the feedback. I do believe it is slightly better to encourage initialization rather than relying on a required default value. It is true that ultimately there is no "undefined" value -- the bits have to be set to some combination of 0s and 1s, and we say that should be `GREEN`. I can change the recommendation to a requirement if people have no objections.